Cargo.toml

```diff
@@ -56,62 +56,136 @@ default = [
 # Force dynamic linking, which improves iterative compile times
 dynamic_linking = ["bevy_dylib", "bevy_internal/dynamic_linking"]
 
-# Optional bevy crates
+# Provides animation functionality
 bevy_animation = ["bevy_internal/bevy_animation"]
+
+# Provides asset functionality
 bevy_asset = ["bevy_internal/bevy_asset"]
+
+# Provides audio functionality
 bevy_audio = ["bevy_internal/bevy_audio"]
+
+# Provides a core render pipeline
 bevy_core_pipeline = ["bevy_internal/bevy_core_pipeline"]
+
+# Plugin for dynamic loading (using [libloading](https://crates.io/crates/libloading))
 bevy_dynamic_plugin = ["bevy_internal/bevy_dynamic_plugin"]
+
+# Adds gamepad support
 bevy_gilrs = ["bevy_internal/bevy_gilrs"]
+
+# [glTF](https://www.khronos.org/gltf/) support
 bevy_gltf = ["bevy_internal/bevy_gltf"]
+
+# Adds PBR rendering
 bevy_pbr = ["bevy_internal/bevy_pbr"]
+
+# Provides rendering functionality
 bevy_render = ["bevy_internal/bevy_render"]
+
+# Provides scene functionality
 bevy_scene = ["bevy_internal/bevy_scene"]
+
+# Provides sprite functionality
 bevy_sprite = ["bevy_internal/bevy_sprite"]
+
+# Provides text functionality
 bevy_text = ["bevy_internal/bevy_text"]
+
+# A custom ECS-driven UI framework built specifically
```

Comment:
```suggestion
# A custom ECS-driven UI framework for bevy
```

Reply:
I would prefer to not write "for Bevy" in a feature of Bevy, but it did end strangely